public/js/utils.js

```diff
@@ -99,26 +99,23 @@ function hidePreloader(d) {
 function getDataFromUrl(url) {
   return axios.get(url)
     .catch( function (error) {
```

Comment:
Oh! One last thing, the argument getting passed is `error` and the function throws `err` so we should update the argument to be `err` or update all the `err` variables to be `error`